crates/iota/src/client_ptb/lexer.rs

```diff
@@ -54,6 +54,31 @@ impl<'l, I: Iterator<Item = &'l str>> Lexer<'l, I> {
         }
     }
 
+    /// Skip the rest of the current shell token (used for comments).
+    fn skip_rest_of_token(&mut self) {
+        let len = self.buf.len();
+        self.offset += len;
+        self.buf = "";
+    }
+
+    /// Skip comment tokens until we hit a command (starts with --) or EOF.
```

Comment:
Maybe I misunderstand but shouldn't we skip until an EOL? Otherwise we can run a commented command?

Reply:
Hm good point, the problem is we can't detect end of lines in the parser since the shell removes this information before
What we could do is to ignore `--` if it comes directly after `//`, but if we have an empty line, so just `// \` and then a command again, it would also ignore this command